OpenKeychain/src/main/res/xml/sync_preferences.xml

```diff
@@ -4,6 +4,11 @@
         android:persistent="false"
         android:title="@string/label_sync_settings_keyserver_title"/>
     <SwitchPreference
+        android:key="enableWifiSyncOnly"
+        android:persistent="true"
+        android:dependency="syncKeyserver"
```

Comment:
Of course....

![screenshot_2016-03-22-13-14-24](https://cloud.githubusercontent.com/assets/11889814/13945077/9d1c6f60-f030-11e5-8a0e-01fb9fbbcc35.png)
![screenshot_2016-03-22-13-14-24](https://cloud.githubusercontent.com/assets/11889814/13945078/9e58b2bc-f030-11e5-8de8-42566633d385.png)